eng/Versions.props

```diff
@@ -11,11 +11,11 @@
   </PropertyGroup>
   <!-- Version number computation -->
   <PropertyGroup>
-    <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
is there a build number or something that comes after this?

Reply:
No, the build number is computed by arcade, This just changes -beta- to -preview-, which is more accurate.

Reply:
This should remain `beta` since you're overriding it directly in FCS.fsproj.

Reply:
fixed